source/opt/types.cpp

```diff
@@ -407,7 +407,7 @@ void RuntimeArray::GetExtraHashWords(std::vector<uint32_t>* words) const {
 
 Struct::Struct(const std::vector<Type*>& types)
     : Type(kStruct), element_types_(types) {
-  for (auto* t : types) {
+  for (const auto* t : types) {
```

Comment:
This looks like an empty loop for NDEBUG.

Reply:
Yes.